internal/storage/paths.go

```diff
@@ -0,0 +1,48 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
This file and others are copied from `storage` module, so they can be used in the new `internal/storage` module